packages/components/src/Spinner/Spinner.tsx

```diff
@@ -1,34 +1,45 @@
-import React from "react";
+import React, { ReactNode } from "react";
 import classnames from "classnames";
 import styles from "./Spinner.css";
+import { BaseSpinner, BaseSpinnerProps } from "./BaseSpinner";
 
-interface SpinnerProps {
+interface SpinnerProps extends BaseSpinnerProps {
   /**
    * Specifies the size of the spinner
    *
-   * @default "base"
+   * @default true
    */
-  readonly size?: "small" | "base";
+  readonly loading?: boolean;
```

Comment:
I get what this prop is doing, but I would get some feedback from other engineers on this name. I might be overthinking it because we were just talking about `Loaders` before this, but getting some end-user thoughts would be nice. 

As an aside, it doesn't look like this has a default but I believe this should default to `false` as I'm wondering if we could get into a state where this component ends up "perma-loading" even once we have content, if the end-user dev doesn't set an explicit "false" when loading is not true (which is hopefully _most_ of the time_). Or is this fine, because it's an optional prop so it's only coming into play when an engineer is explicitly using it?

(No specific "change requested" here, just some ideas that might be worth discussing)